lib/Weathermap/Integrations/Cacti/WeatherMapCacti10ManagementPlugin.php

```diff
@@ -59,13 +59,13 @@ public function handleManagementMainScreen($request, $appObject)
         $locale = $this->manager->application->getLocale();
         print "<h3>This is the React UI below here</h3>";
         print "<h1>INCOMPLETE</h1>";
-        print '<style>@import "cacti-resources/mgmt/main.css";</style>';
+        print '<style>@import "/cacti/plugins/weathermap/cacti-resources/mgmt/main.css";</style>';
         print "<h3>Testing API</h3>";
         print '<a href="?action=listmaps">Map List</a> - ';
         print '<a href="?action=app_settings">Settings</a> - ';
         print '<a href="?action=listmapfiles">Map Files</a>';
         print "<div id='weathermap-mgmt-root' data-locale='" . $locale . "' data-url='" . $this->makeURL(array("action" => "app_settings")) . "'></div>";
-        print '<script type="text/javascript" src="cacti-resources/mgmt/main.js"></script>';
+        print '<script type="text/javascript" src="/cacti/plugins/weathermap/cacti-resources/mgmt/main.js"></script>';
```

Comment:
applying the above` global $config`
`        print '<script type="text/javascript" src="' . $config['url_path'] . 'plugins/weathermap/cacti-resources/mgmt/main.js"></script>';
`

Reply:
Should be done now.